R/summarize_scores.R

```diff
@@ -44,6 +47,14 @@ summarize_risk <- function(data, weights = .risk_weights) {
   risk
 }
 
+#' @export
+summarize_scores.list <- function(data, weights = .risk_weights) {
+  weights <- weights[which(names(weights) %in% names(data))]
+  weights <- weights / sum(weights, na.rm = TRUE)
+
+  1 - sum(as.numeric(data[names(weights)]) * weights, na.rm = TRUE)
```

Comment:
Sorry one last thing, I think we should return the score here and not the inverse of it. Now that we have renamed from summarize risk to summarize score, my intuition is that a higher scoring package would be better.

Reply:
Good catch, I was thinking the same thing. I'll opt not to invert it now that we're not calculating "risk".